.gitpod.yml

```
@@ -2,4 +2,13 @@ tasks:
   - init: |
       make init
     command: |
-      make dev-node/start
\ No newline at end of file
+      make dev-node/start
+  - init: |
+      yarn --cwd interface
```

Possibly this line is not needed
```suggestion
      yarn --cwd interface
```